spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Fusion.scala

```diff
@@ -97,6 +100,15 @@ private[mkldnn] object Fusion {
     } else node
   }
 
+  private def findNext(node: Node[AbstractModule[Activity, Activity, Float]])
+  : Seq[Node[AbstractModule[Activity, Activity, Float]]] = {
+    if (node.element.isInstanceOf[Identity]) {
+      node.nextNodes.flatMap(n => findNext(n))
+    } else {
+      Seq(node)
+    }
+  }
+
```

Comment:
May I ask the logics of this method?
We are trying to find next nodes of the node passed in as an argument, 
if it is Identity Node, return the next nodes of it's next nodes
else return the node as Sequence.
The logic is kind of unclear.

Reply:
If current node is not `Identity`, it will return immediately. Otherwise, it will find the next recursively. I use a `flatMap` to flat all arrays to a single seq here.